quipucords/api/common/entities.py

```diff
@@ -82,6 +106,16 @@ def mac_addresses(self):
         with suppress(TypeError):
             return json.loads(self._fingerprints.mac_addresses)
 
+    @property
+    def provider_id(self):
+        """Retrieve provider_id.
+
+        Relying on fingerprint name is wrong for at least some cases. On AWS EC2 vms,
+        this should be the content of /var/lib/cloud/data/instance-id (which would be a
+        a new fact).
+        """
+        return self._fingerprints.name
+
```

Comment:
Here I'm repeating the behaviour implemented on 1.0.1 release, BUT now we know this is probably wrong.
Do you think it is better removing this fact @justinkreft @gtanzillo ?

the only advantage I see on keeping it is that maintaining this field will send provider type information to hbi (for instance "aws"), but IDK if this is useful for HBI.

Reply:
Based on our slack conversation, and the fact that `ProviderId` is described this way in the [API spec](https://github.com/RedHatInsights/insights-host-inventory/blob/b7a0b529713d31ebfa5b668a3c8df4b7b624be01/swagger/api.spec.yaml#L922-L929), I would lean toward removing it. It seems that the current behavior will insert a hostname in most cases, but that is pretty much useless for the purpose of `provider_id`, as it's defined in the API spec.